### DISCUSSION TITLE
stopped working

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [community/community#10724] [Copilot Conversations] [answered]
It worked, but today the icon at the bottom right doesn't appear and when I press CTRL + enter it just creates a newline.

The extension is enabled and installed. And I have access to copilot.

I cannot see any errors or something

extension version:
v1.7.4760

**Comment by 1Euro7Cent** [comment, marked as answer]
Ok. It was my VS Code. It just didn't load any extension even after VS Code reload. After closing and opening it back up it fixed itself.

**Comment by infernal-pepe** [comment]
using pycharm and I've restarted project a few times and it's still not working.  just keeps popping up chat, copilot disappeared.  dunno why I'm paying for something that isn't working.
![image](https://github.com/community/community/assets/145865841/993badcc-69ea-46ce-9edd-8d0479d7630e)

**Comment by a7medelbatal** [comment]
It just didn't load any extension, so after close vs code and opening it back up it fixed itself yeah.
